playbooks/common/openshift-master/config.yml

```diff
@@ -275,11 +267,29 @@
                                       | map(attribute='stdout')
                                       | list) }}"
 
+- name: Configure master named certificates
+  hosts: oo_masters_to_config
+  vars:
+    named_certs_specified: "{{ openshift_master_named_certificates is defined }}"
+  tasks:
+  - name: Land named certificates
+    copy: src="{{ item.certfile }}" dest="{{ master_cert_config_dir }}"
+    with_items: openshift_master_named_certificates
+    when: named_certs_specified | bool
+  - name: Land named certificate keys
+    copy: src="{{ item.keyfile }}" dest="{{ master_cert_config_dir }}"
+    with_items: openshift_master_named_certificates
+    when: named_certs_specified | bool
```

Comment:
So, here is a kicker...  we don't necessarily want to fail if the cert files aren't present for subsequent runs (Think a second run of the installer to change configuration, upgrade, etc).

I wonder if there should also be a when statement on these that tests that the local file exists first...  Or possibly conditionally fail if the files are not present, so that we can trigger the failure on initial install with the wrapper and not fail for subsequent runs... ?